src/main/java/uk/gov/digital/ho/hocs/casework/api/dto/GetCaseSummaryResponse.java

```diff
@@ -8,12 +8,7 @@
 import uk.gov.digital.ho.hocs.casework.domain.model.CaseSummary;
 
 import java.time.LocalDate;
-import java.util.ArrayList;
-import java.util.Comparator;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
Up to you, but probably best to change these to non wild card imports. For the sonar check... my IDE does this automatically, it's so annoying

Reply:
You can set that setting in Intellij to 999 and never have it cause an issue again 🙂